### ISSUE TITLE
2025-11-12: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#415]
# 每日更新报告（2025-11-12）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-12 23:56:47 | Canlin Guo | [Refactor] Remove redundant TP gather/split in split_qkv in QwenVL (#28271) |
| 2025-11-12 23:51:43 | Alexander Matveev | [Performance][Hopper] Avoid M dim padding to 4x for most cases (due to cuda graphs paddings) (#28492) |
| 2025-11-12 22:13:41 | Harry Mellor | Fix pre-commit (and XPU) on `main` (#28556) |
| 2025-11-12 22:03:52 | wangxiyuan | [V0 deprecation] Deprecate use_v1 parameter (#28112) |
| 2025-11-12 21:02:06 | Jee Jee Li | [Bugfix] Fix gpt_oss packed_modules_mapping (#28536) |
| 2025-11-12 21:01:14 | TJian | [ROCm] [Bugfix] Fix `fused_qknorm_rope_kernel` rocm compatibility (#28500) |
| 2025-11-12 19:53:08 | ZhengHongming888 | add cpu option for p/d in nixl_connector (#28356) |
| 2025-11-12 18:44:58 | ziruiliu | [KVConnector] Enable get_block_ids_with_load_errors() in LMCache connector  (#27978) |
| 2025-11-12 18:20:55 | Chaojun Zhang | [XPU]Fix crash due to removed VLLM_USE_V1 attribute (#28520) |
| 2025-11-12 18:14:00 | wuyaoxuehun | [Model] fix glm4_moe_mtp load weights with GLM-4.6 checkpoint. (#27597) |
| 2025-11-12 17:03:56 | yyzxw | [BugFix]: --enable-lora with model granite-4.0-micro crash (#27733) |
| 2025-11-12 16:51:19 | Huamin Li | [CI] Introduce autorun_on_main feature (#27836) |
| 2025-11-12 15:09:33 | Chenguang Zheng | [CI/Build] Fix crash due to removed VLLM_USE_V1 attribute in EPD (#28521) |
| 2025-11-12 13:03:01 | Lukas Geiger | [Core] Cache `vllm_is_batch_invariant` (#28304) |
| 2025-11-12 13:00:45 | ai-jz | [Benchmark] Add retry support to fix workload bias in multi-turn benchmark (#28493) |
| 2025-11-12 11:13:21 | Fanli Lin | [BugFix] Add fallback path in `apply_rotary_pos_emb_flashattn` for non-cuda platforms (#28447) |
| 2025-11-12 10:58:33 | Chenguang Zheng | [Core] Encoder separation for Encode-Prefill-Decode Disaggregation (#25233) |
| 2025-11-12 10:55:10 | Lukas Geiger | [Model][Qwen3VL] Simplify `get_mrope_input_positions` using numpy (#28302) |
| 2025-11-12 10:34:36 | Andreas Karatzas | `VLLM_USE_TRITON_FLASH_ATTN` V0 variable deprecation (#27611) |
| 2025-11-12 09:43:06 | Li, Jiang | [CPU] Refactor CPU attention backend (#27954) |
| 2025-11-12 09:22:16 | wangxiyuan | [[V0 deprecation]]Remove VLLM_USE_V1 env (#28204) |
| 2025-11-12 08:49:29 | Isotr0py | [Performance] Cache loaded custom logitsprocs to avoid overheads (#28462) |
| 2025-11-12 08:46:18 | Yanan Cao | [Frontend] Change CompilationMode to a proper Enum (#28165) |
| 2025-11-12 08:41:54 | Ilya Markov | [BugFix] Graceful handling of torch symm mem errors. (#27671) |
| 2025-11-12 08:34:58 | Adrian Abeyta | Use FLASHINFER MLA backend when testing fp8_kv_scale_compile (#28491) |
| 2025-11-12 07:32:20 | Max Hu | [Feature] Allow configuring FlashInfer workspace size (#28269) |
| 2025-11-12 07:16:12 | Michael Goin | [Bugfix] Disable shared expert overlap if Marlin MoE is used (#28410) |
| 2025-11-12 06:53:59 | Michael Goin | Add Zurich vLLM Meetup (#28488) |
| 2025-11-12 06:28:07 | wangxiyuan | [Misc] Cleanup Executor interface (#28441) |
| 2025-11-12 03:16:47 | Kyuyeun Kim | [TPU] Rename path to tpu platform (#28452) |
| 2025-11-12 03:06:21 | Jee Jee Li | [LoRA][1/N]Remove LoRA extra vocab (#28382) |
| 2025-11-12 02:59:16 | Jie Luo | [Misc] fix typo in DCP comment (#28389) |
| 2025-11-12 02:57:12 | Canlin Guo | [V0 deprecation] Clean up num_prefill_tokens logic for V0 (#28203) |
| 2025-11-12 02:28:47 | Jialin Ouyang | [Perf] Use np.ndarray instead of list[list[int]] to reduce GC overhead (#28245) |
| 2025-11-12 02:26:04 | Lukas Geiger | [Misc] Remove unused attention prefix prefill ops functions (#26971) |
| 2025-11-12 02:18:02 | Fanli Lin | [BugFix] Fix Siglip2Attention on XPU (#28448) |
| 2025-11-12 02:02:23 | Xin Yang | [Kernel] Optimize rms_norm kernel (#27931) |
| 2025-11-12 01:13:51 | Matthew Bonanni | Prefer FlashAttention MLA as default over FlashMLA (#27363) |
| 2025-11-12 01:09:47 | Zhewen Li | [CI/Build] Refactor Attention backend for test_prefix_prefill from xformers to SDPA (#28424) |
| 2025-11-12 01:06:00 | xuebwang-amd | [Quantization] fix attention quantization of gpt_oss model (#27334) |
| 2025-11-12 01:05:22 | xuebwang-amd | [ROCm][Quantization] extend AMD Quark to support mixed-precision quantized model (#24239) |
| 2025-11-12 01:00:31 | zhrrr | [Kernel][Perf] fuse QK Norm and RoPE into one cuda kernel for Qwen Model (#27165) |
| 2025-11-12 00:57:43 | Nicolò Lucchesi | [NIXL] Generalize block-first backend layouts (FlashInfer-like) (#28282) |
| 2025-11-12 00:46:04 | Michael Goin | Remove weight_scale.T special case for SM90 Block FP8 CUTLASS kernel (#28431) |
| 2025-11-12 00:45:49 | the-codeboy | [Doc] Fix typo in serving docs (#28474) |
| 2025-11-12 00:01:30 | usberkeley | [BugFix] Add test_outputs.py to CI pipeline (#28466) |
